src/app/views.py

```diff
@@ -1291,30 +1291,32 @@ def archiveRequests(request, license_id=None):
     returns archive_requests.html template
     """
     context_dict = {}
+    if request.user.is_authenticated():       
+        user = request.user
+        github_login = user.social_auth.get(provider='github')
+        if utils.checkPermission(user):    
+            archive = request.POST.get('archive', True)
+            license_id = request.POST.get('license_id', False)
```

Comment:
This will error out for `GET` request since request.POST will not exist, not sure though

Reply:
Also, we don't need these lines, not used anywhere
```
archive = request.POST.get('archive', True)
license_id = request.POST.get('license_id', False)
```